src/App.vue

```diff
@@ -54,14 +54,85 @@
         'Functionele en analytische cookies accepteren',
         'Alleen functionele cookies',
       ],
+      /*       geojson_data: {
+        "type": "FeatureCollection",
+        "features": [
+          {
+            "type": "Feature",
+            "properties": {
+              "locationId": "NL02L10a",
+              "value": 4,
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                6.235080651990514,
+                53.016662661129416,
+              ],
+            },
+          },
+          {
+            "type": "Feature",
+            "properties": {
+              "locationId": "NL12_210",
+              "value": 3,
+            },
+            "geometry": {
+              "type": "Point",
+              "coordinates": [
+                4.841526304081336,
+                52.55228533949656,
+              ],
+            },
+          },
+        ],
+      }, */
+      layers: [],
+
     }),
-    computed: {
-      ...mapState({
-        wmsLayers: ({ map }) => map.wmsLayers,
+    computed: { 
+      ...mapState({ 
+        testData: (state) => state.testData,
       }),
     },
+    watch: {
+      testData() {
```

Comment:
Getting and and adding initial data to the map can be moved to a Vuex action. Then we just need to call something like `getInitialMapData` once on `created` or `mounted`.